kalite/distributed/features/steps/generic.py

```diff
@@ -1,6 +1,9 @@
 from behave import *
 from kalite.testing.behave_helpers import *
+from django.contrib.auth.models import User
 
 @given("I am on the homepage")
 def step_impl(context):
-    go_to_homepage(context)
\ No newline at end of file
+	if not User.objects.exists():
```

Comment:
I consider this unexpected behavior for this step. I would create a new step like this:
```python
@given("There is a superuser")
def step_impl(context):
    if not User.objects.exists():
        User.objects.create_superuser(username='superusername', password='superpassword', email='super@email.com')
```